### ISSUE TITLE
RdRp_HMM_profile.db.h3?

### ISSUE THREAD

**Issue by dawnmy** [JustineCharon/RdRp-scan#1]
Thank you for developing this useful data resource. I could not find RdRp_HMM_profile.db.h3? in Profile_db_and_alignments, but RdRp_HMM_profile_CLUSTALO.db.h3? files are there. Is this because of the typo?

**Comment by JustineCharon** [comment]
Hi,

Yes it's the same file, I just added Clustalo because those profiles have been obtained using ClustalOmega.

Please let me know if any further info is need,

Kind regards

Justine

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
From: Z.-L. Deng ***@***.***>
Sent: Saturday, October 8, 2022 3:28:29 PM
To: JustineCharon/RdRp-scan ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [JustineCharon/RdRp-scan] RdRp_HMM_profile.db.h3? (Issue #1)


Thank you for developing this useful data resource. I could not find RdRp_HMM_profile.db.h3? in Profile_db_and_alignments, but RdRp_HMM_profile_CLUSTALO.db.h3? files are there. Is this because of the typo?

**Comment by xsq2022** [comment]
hi~ thanks for your great resource!
 I want to check some hypothetical protein  to see if they are RDRP-like protein by using ` hmmscan --domtblout hmmout  --domE 1e-5 RdRp_HMM_profile_CLUSTALO.db  hypothetical_protein.orf.fa` 
I downloaded all four files (RdRp_HMM_profile.db.h3?) in **[Profile_db_and_alignments]** 
But it seem that a `RdRp_HMM_profile_CLUSTALO.db.hmm` file lost . this **hmmscan --domtblout hmmout** step cannot work .
how can i get this file ? 
thanks a lot

**Comment by JustineCharon** [comment]
Hi,

Could you run it again using "RdRp_HMM_profile_CLUSTALO.db" as the name for HMM-profile database ? I have added the "CLUSTALO" mention in the name as some additional alignment method will be added in future... I encountered issues running --domE option in the past, maybe try without this option and please let me know if it works ;)

**Comment by xsq2022** [comment]
I tried，but it failed when I used it in hmmscan.---- Replied Message ----FromJustine ***@***.***>Date10/12/2022 22:03 ***@***.***> ***@***.******@***.***>SubjectRe: [JustineCharon/RdRp-scan] RdRp_HMM_profile.db.h3? (Issue #1)
Hi,
Could you run it again using "RdRp_HMM_profile_CLUSTALO.db" as the name for HMM-profile database ? I have added the "CLUSTALO" mention in the name as some additional alignment method will be added in future... I encountered issues running --domE option in the past, maybe try without this option and please let me know if it works ;)

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/JustineCharon/RdRp-scan/issues/1#issuecomment-1276237847",
"url": "https://github.com/JustineCharon/RdRp-scan/issues/1#issuecomment-1276237847",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by JustineCharon** [comment]
Ok I will try it on my side and keep you posted...
Sorry for the bug !

**Comment by xsq2022** [comment]
That's all right. Thank you. Looking forward to it.！---- Replied Message ----FromJustine ***@***.***>Date10/13/2022 20:11 ***@***.***> ***@***.******@***.***>SubjectRe: [JustineCharon/RdRp-scan] RdRp_HMM_profile.db.h3? (Issue #1)
Ok I will try it on my side and keep you posted...
Sorry for the bug !

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/JustineCharon/RdRp-scan/issues/1#issuecomment-1277512181",
"url": "https://github.com/JustineCharon/RdRp-scan/issues/1#issuecomment-1277512181",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by dawnmy** [comment]
@xsq2022 @JustineCharon 
I am able to run it without any issue:
```
hmmscan --cpu {threads} -E 1e-5 --domtblout {output.domain_out} \
            --tblout {output.seq_out} {params.db} {input}
```

**Comment by luoqingqing777** [comment]
@dawnmy 
what did you put as {params.db} ? 
i put  `RdRp_HMM_profile_CLUSTALO.db` as {params.db} ,it can't  work

**Comment by dawnmy** [comment]
I used `Profile_db_and_alignments/RdRp_HMM_profile_CLUSTALO.db` as the `params.db`. Which version of HMMER did you use? Mine is HMMER 3.2.1.
